### ISSUE TITLE
flok-web  cannot find module - yarn and npm

### ISSUE THREAD

**Issue by loopyNid** [munshkr/flok#129]
Hello,
I have been trying to run the flok-web server. But both installing it via npm or compiling via yarn I get an error.

via npm:

```
root@flokDev:~# flok-web 
next-transpile-modules - DEPRECATED - fallbacking to previous module resolution system for module "lib0", you can now just pass the name of the package to transpile and it will detect its real path without you having to pass a sub-module.
Error: failed to load next.config.js, see more info here https://nextjs.org/docs/messages/next-config-error
/usr/lib/node_modules/flok-web/node_modules/next-transpile-modules/src/next-transpile-modules.js:173
          throw new Error(
                ^

Error: next-transpile-modules - an unexpected error happened when trying to resolve "lib0". Are you sure the name module you are trying to transpile is correct, and it has a "main" or an "exports" field?
Error: Can't resolve 'lib0' in '/root'
    at getPackageRootDirectory (/usr/lib/node_modules/flok-web/node_modules/next-transpile-modules/src/next-transpile-modules.js:173:17)
    at Array.map (<anonymous>)
    at withTM (/usr/lib/node_modules/flok-web/node_modules/next-transpile-modules/src/next-transpile-modules.js:185:34)
    at Object.<anonymous> (/usr/lib/node_modules/flok-web/next.config.js:16:18)
    at Module._compile (node:internal/modules/cjs/loader:1101:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1153:10)
    at Module.load (node:internal/modules/cjs/loader:981:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at Module.require (node:internal/modules/cjs/loader:1005:19)
    at require (node:internal/modules/cjs/helpers:102:18)

```

and via yarn:
```
root@flokDev:~/flok/packages/web/bin# nodejs flok-web.js 

Command 'nodejs' not found, but can be installed with:

apt install nodejs

root@flokDev:~/flok/packages/web/bin# node flok-web.js 
node:internal/modules/cjs/loader:361
      throw err;
      ^

Error: Cannot find module '/root/flok/node_modules/flok-core/lib/index.js'. Please verify that the package.json has a valid "main" entry
    at tryPackage (node:internal/modules/cjs/loader:353:19)
    at Function.Module._findPath (node:internal/modules/cjs/loader:566:18)
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:919:27)
    at Function.Module._load (node:internal/modules/cjs/loader:778:27)
    at Module.require (node:internal/modules/cjs/loader:1005:19)
    at require (node:internal/modules/cjs/helpers:102:18)
    at Object.<anonymous> (/root/flok/packages/web/lib/Server.js:11:20)
    at Module._compile (node:internal/modules/cjs/loader:1101:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1153:10)
    at Module.load (node:internal/modules/cjs/loader:981:32) {
  code: 'MODULE_NOT_FOUND',
  path: '/root/flok/node_modules/flok-core/package.json',
  requestPath: 'flok-core'
}
```

I don't know what i could be doing wrong.
Thank you.

**Comment by munshkr** [comment]
Hi, yes I think I broke something when pushing the last versions (0.4.6 and 0.4.7), both were published yesterday in a rush. In the mean time, please try downgrading to 0.4.5:

```npm install -g flok-web@0.4.5```

If you are using the repository, do `git checkout v0.4.5` and build the web package again.

**Comment by vin-ni** [comment]
I think it should be npm install -g flok-web@0.4.4 for now. 0.4.5 is somehow gone

**Comment by munshkr** [comment]
This has finally been fixed on version 0.4.8. Feel free to try upgrading: `npm -g install flok-web@0.4.8`
